pkgs/development/interpreters/python/wrap.sh

```diff
@@ -30,7 +30,7 @@ patchPythonScript() {
 
     # The magicalSedExpression will invoke a "$(basename "$f")", so
     # if you change $f to something else, be sure to also change it
-    # in pkgs/top-level/python-packages.nix!
+    # in wrap-python.nix!
     # It also uses $program_PYTHONPATH.
     sed -i "$f" -re '@magicalSedExpression@'
 }
```

Comment:
Could you elaborate some more?  My intention was only to fix the broken reference to `wrap-python.nix`, nothing else.  If there's more to be done in `wrap.sh`, it should probably not be in this PR, since fixing that reference is already extraneous to this PR, which is about bitmessage.